bin/ch_misc.c

```diff
@@ -406,6 +406,19 @@ void log_ids(const char *func, int line)
    }
 }
 
+void logging_print(bool fail) {
+   TRACE("trace");
+   DEBUG("debug");
+   VERBOSE("verbose");
+   INFO("info");
+   WARNING("warning");
```

Comment:
Add `ERROR`? Do we have `ERROR` in the C stuff?

Reply:
The C code doesn't have an `ERROR` logging function.
```
$ misc/grep -F '#define FATAL'
./bin/ch_misc.h:68:#define FATAL(...)   msg_fatal(      __FILE__, __LINE__, 0, __VA_ARGS__);
$ misc/grep -F '#define ERROR'
$
```